2-projects/1-todo-list/script.js

```diff
@@ -17,9 +55,23 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let todoInputEl = document.querySelector('#todoInput').value; // Reads the value of the #todoInput element
+  let todoObj = { task: todoInputEl, completed: false }; // Assigns a new task to an object
+  todos.push(todoObj); // Pushes the todoObj to global todos array
+  let newTodo = [todoObj]; // Adds todoObj to an array
+  populateTodoList(newTodo); // This function is called now with the newTodo as the parameter
```

Comment:
Instructions say 

> You will want to remove the ones in the current HTML after you have created them using JavaScript

I think what it wants you to do is call

`populateTodoList(todos)`

and pass it the full list every time.

Then the populateTodoList function will clear the list (completely empty) and rebuild it with the full list